Userland/Libraries/LibC/stdio.cpp

```diff
@@ -1159,10 +1191,16 @@ void dbgputstr(const char* characters, size_t length)
     syscall(SC_dbgputstr, characters, length);
 }
 
-char* tmpnam(char*)
+char* tmpnam(char* s)
 {
-    dbgln("FIXME: Implement tmpnam()");
-    TODO();
+    static char tmpnam_buffer[L_tmpnam];
```

Comment:
Please use an `Array` here, in the C++ files we are not bound by C's limitations.

Reply:
Okay. I was hesitant about using these things in stdio, but I will change them right away.